int8 cmd
Point pointInfo
